ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/CloudSourceIntegrations/OcmIntegrationForm.tsx

```diff
@@ -53,16 +53,14 @@ export const validationSchema = yup.object().shape({
         }),
         skipTestIntegration: yup.bool(),
     }),
-    updatePassword: yup.bool(),
+    updateCredentials: yup.bool(),
```

Comment:
Does PaladinCloudIntegrationForm.tsx file also need this change?

Reply:
Yes good catch, thanks.